src/components/ArticleItem.jsx

```diff
@@ -25,6 +29,41 @@ function ArticleItem() {
         
     }, []);
 
+    const updateArticleVote = (direction, votes) => {
+        let votesInt = parseInt(votes)
+        setArticle(()=> {
+            return {...article, votes: 
+                direction === "increment" 
+                ?
+                votesInt + 1
+                :
+                votesInt - 1
+            
+            }
+        })
+        patchArticleVotes(article_id, direction)
+        .catch((err) => {
+            setErrors((previousState)=> {
+                return [
+                    ...previousState,
+                    {
+                        id: previousState + 1,
+                        message: err.response.data.message,
```

Comment:
```js
                        message: err.response.data.message,
```
This works perfectly for your votes but seeing as this looks like something you're going to re-use consider the situation when you don't get a response body. e.g. No network connection. If you want to add something in to handle that now or deal with it in a later ticket that's fine.

You can turn it off in the browsers dev tools under network > throttling FYI